DfE.FindInformationAcademiesTrusts/Services/ExportService.cs

```diff
@@ -51,16 +52,27 @@ public byte[] ExportAcademiesToSpreadsheetUsingProvider(Data.Trust trust, Trust.
                     IsOfstedRatingBeforeOrAfterJoining(currentRating?.OfstedRatingScore ?? OfstedRatingScore.None, academy.DateAcademyJoinedTrust, currentRating?.InspectionDate),
                     currentRating?.InspectionDate?.ToString(StringFormatConstants.ViewDate) ?? string.Empty,
                     academy.PhaseOfEducation ?? string.Empty,
+                    ageRangeString,
                     academy.NumberOfPupils?.ToString() ?? string.Empty,
                     academy.SchoolCapacity?.ToString() ?? string.Empty,
                     academy.PercentageFull.HasValue ? $"{academy.PercentageFull}%" : string.Empty,
-                    academy.PercentageFreeSchoolMeals.HasValue ? $"{academy.PercentageFreeSchoolMeals}%" : string.Empty
+                    academy.PercentageFreeSchoolMeals.HasValue ? $"{academy.PercentageFreeSchoolMeals}%" : string.Empty,
                 ];
             });
 
             return GenerateSpreadsheet(trustSummary, academies, headers, dataExtractor);
         }
 
+        private static string GetAgeRangeString(Data.Academy academy)
+        {
+            if (academy.AgeRange is not null)
```

Comment:
`academy.AgeRange` is not nullable so this `if` is not required

Reply:
We could remove this method altogether and just use `$"{academy.AgeRange.Minimum} - {academy.AgeRange.Maximum}"`